openai-java-core/src/main/kotlin/com/openai/models/chat/completions/StructuredChatCompletion.kt

```diff
@@ -0,0 +1,169 @@
+package com.openai.models.chat.completions
+
+import com.openai.core.JsonField
+import com.openai.core.JsonValue
+import com.openai.errors.OpenAIInvalidDataException
+import com.openai.models.chat.completions.ChatCompletion.Choice.FinishReason
+import com.openai.models.chat.completions.ChatCompletion.Choice.Logprobs
+import com.openai.models.chat.completions.ChatCompletion.ServiceTier
+import com.openai.models.completions.CompletionUsage
+import java.util.Objects
+import java.util.Optional
+
+class StructuredChatCompletion<T : Any>(
+    val responseFormat: Class<T>,
+    val chatCompletion: ChatCompletion,
```

Comment:
```suggestion
    @get:JvmName("responseFormat") val responseFormat: Class<T>,
    @get:JvmName("rawChatCompletion") val rawChatCompletion: ChatCompletion,
```

Otherwise I think Kotlin will generate `getResponseFormat()` and `getChatCompletion()` methods, instead of non-`get` prefixed